test/test_autograd.py

```diff
@@ -6607,7 +6616,7 @@ def f(x, y):
 
         # output unrelated to all inputs
         def f(x, y):
-            return torch.randn([])
+            return torch.ones([])
```

Comment:
Why this change?
Or is it just that randomness is never working with vmap? Would a `randn_like(x)` be better?

Reply:
Yep, it is because randomness does not work with vmap